src/structure/3dmol/index.d.ts

```diff
@@ -475,10 +511,16 @@ export interface PresetElementColors {
 
 export declare const elementColors: PresetElementColors;
 
-declare namespace Parsers {
-    function assignBonds(atoms: Array<Partial<AtomSpec>>): void;
-}
-
 export declare class Vector3 {
     constructor(x: number, y: number, z: number);
+    public normalize(): Vector3;
+}
```

Comment:
Why declare a Vector3 class? Often you are using a three vector across the package, why does it necessitate a class here?

Reply:
This is declaring a class which exists in 3Dmol, to be able to use it with typescript